src/Kovan-DssSpell.sol

```diff
@@ -16,36 +16,275 @@
+            INPUT_CONDUIT: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            OUTPUT_CONDUIT: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            URN:  0xf22C7F5A2AecE1E85263e3cec522BDCD3e392B59,
+            gemID: "RWA004",
+            joinID: "MCD_JOIN_RWA004_A",
+            urnID: "RWA004_A_URN",
+            inputConduitID: "RWA004_A_INPUT_CONDUIT",
+            outputConduitID: "RWA004_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA004",
+            ilk: "RWA004-A",
+            ilk_string: "RWA004-A",
+            ilkRegistryName: "RWA004-A: Centrifuge: Harbor Trade Credit",
+            RATE: SEVEN_PCT,
+            CEIL: 7 * MILLION,
+            PRICE: 8_815_730 * WAD,
+            MAT: 11_000,
+            TAU: 0,
+            DOC: "QmYR2PXwLpdXS8Vp1yS39SPFT1XhmgbsK6XvZ9ApRpNV8M"
+        });
+
+        // https://vote.makerdao.com/polling/QmZCPDVY#poll-detail
+        CentrifugeCollateralValues memory RWA005 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x1233d0DBb55A4Bb41D711d4B584f8DDB15A2Ff88,
+            GEM: 0xcB2A48D26970eE7193d66BAc6F1b3090f2E8f82B,
+            OPERATOR: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            INPUT_CONDUIT: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            OUTPUT_CONDUIT: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
+            URN:  0xdB9f0700EbBac596CCeF5b14D5e23664Db2A184f,
+            gemID: "RWA005",
+            joinID: "MCD_JOIN_RWA005_A",
+            urnID: "RWA005_A_URN",
+            inputConduitID: "RWA005_A_INPUT_CONDUIT",
+            outputConduitID: "RWA005_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA005",
+            ilk: "RWA005-A",
+            ilk_string: "RWA005-A",
+            ilkRegistryName: "RWA005-A: Centrifuge: Fortunafi",
+            RATE: FOUR_PT_FIVE_PCT,
+            CEIL: 15 * MILLION,
+            PRICE: 17_199_394 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmbgDoPn6UcfSDENDqHLgatMFoqXikC8E8it9WaZXyLXmc"
+        });
+
+        // https://vote.makerdao.com/polling/QmdHV2tW#poll-detail
+        CentrifugeCollateralValues memory RWA006 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x039B74bD0Adc35046B67E88509900D41b9D95430,
+            GEM: 0x4E65F06574F1630B4fF756C898Fe02f276D53E86,
+            OPERATOR: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            INPUT_CONDUIT: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            OUTPUT_CONDUIT: 0x652A3B3b91459504A8D1d785B0c923A34D638218,
+            URN:  0x6fa6F9C11f5F129f6ECA4B391D9d32038A9666cD,
+            gemID: "RWA006",
+            joinID: "MCD_JOIN_RWA006_A",
+            urnID: "RWA006_A_URN",
+            inputConduitID: "RWA006_A_INPUT_CONDUIT",
+            outputConduitID: "RWA006_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA006",
+            ilk: "RWA006-A",
+            ilk_string: "RWA006-A",
+            ilkRegistryName: "RWA006-A: Centrifuge: Alternative Equity Advisers",
```

Comment:
Did this company change their name or something? This doesn't match the name in the poll ("Peoples Company").

Reply:
You can see Alternative Equity Advisers in the risk assessment https://forum.makerdao.com/t/p1-drop-mip6-risk-assessment-alternative-equity-advisors-drop-us-agricultural-real-estate/8232 Peoples Company is the asset originator, while Alternative Equity Advisers is the issuer. Alternative Equity Advisers is the name that should be used externally, hence why I changed it there.